EIPS/eip-cap-gas-limit.md

```diff
@@ -0,0 +1,54 @@
+---
+eip: <to be assigned>
+title: Cap Gas Target
+description: Set an in-protocol cap for the gas target
+author: lightclient (@lightclient)
+discussions-to: <URL>
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-08-21
+---
+
+## Abstract
+
+Sets an in-protocol cap for the gas target.
+
+## Motivation
+
+High gas target increases pressure on the network. In the benign case, it
+increases the state and history faster than we can sustain. In the malicious
+case, it amplifies the devestation of certain denial-of-service attacks.
+
+## Specification
+
+As of the fork block `N`, consider blocks with a `gas_target` greater than
+`15,000,000` invalid.
+
+## Rationale
+
+### Why Cap the Gas Target
+
+The gas target is currently under the control of block proposers. They have the
```

Comment:
The gas target isn't under their control, the gas limit is.

Reply:
Silly me.